tests/test_issue_10926_watcher_pid_reuse_guard.py

```
@@ -0,0 +1,497 @@
+    }
+
+
+def run_shell(
+    shell_argv: list[str],
+    script: str,
+    args: list[str],
+    env: dict[str, str],
+    timeout: float = 15.0,
+    pass_fds: tuple[int, ...] = (),
+) -> subprocess.CompletedProcess:
+    return subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)